packages/profiling-node/bindings/cpu_profiler.cc

```diff
@@ -111,14 +111,13 @@ class MeasurementsTicker {
 
   size_t listener_count();
 
-  ~MeasurementsTicker() {
+  void Cleanup() {
     uv_timer_stop(&timer);
```

Comment:
This returns an error message, let's make sure that it is actually successfully stopped